app/views/dashboard_v2/dashboard.html.erb

```diff
@@ -0,0 +1,87 @@
+<%# TODO: Topics to display if the user is new and not yet subscribed to other posts %>
+<div class="container">
+  <div class="row">
+    <%# Topics %>
+    <div class="col-lg-8">
+      <%= render partial: "dashboard_v2/blog"%>
+      <%= render partial: "dashboard_v2/topicCard"%>
+    </div>
+    <%# Sidebar %>
+    <div class="col-lg-4">
+      <%# Admin feature %>
+      <%= feature('dashboard-feature') %>
+      <%= render partial: "sidebar/dashboard_v2"%>
+    </div>
+  </div>
+  <%# Additional Topics %>
+  <div class="other-topics">
+    <h4>
+      <i class="form-grey d-none d-lg-inline"><%= translation('dashboard_v2.sidebar.interest_topics')%></i>
+    </h4>
+    <%# TODO Recommended topics %>
```

Comment:
I am not sure which query to use to obtain recommended topics

Reply:
this should be "featured" i believe!

Reply:
On the mockup, it shows `recommended`.... should it be changed to featured?
![Screenshot 2021-01-06 at 20 41 17](https://user-images.githubusercontent.com/7622875/103802034-99f4bb00-505f-11eb-9f41-ee77cb08e427.png)

Reply:
Yes that sounds good. I think the mockup is just older and didn't get updated to the latest wording. Thanks!!

Reply:
Great! Thanks!